eth/tracers/native/noop.go

```diff
@@ -35,7 +36,7 @@ func init() {
 type noopTracer struct{}
 
 // newNoopTracer returns a new noop tracer.
-func newNoopTracer(ctx *tracers.Context, _ json.RawMessage) (*tracers.Tracer, error) {
+func newNoopTracer(ctx *tracers.Context, _ json.RawMessage, _ *params.ChainConfig) (*tracers.Tracer, error) {
```

Comment:
Please don't use unnamed params. It  was an unfortunate language design that the authors also think is bad now but can't revert it. Still, live code should not use it, it just makes things harder to understand and later modify if the param is ever needed.